packages/samples/headless-react/project.json

```diff
@@ -1,3 +1,18 @@
 {
-  "name": "headless-react-samples"
+  "name": "headless-react-samples",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "targets": {
+    "cached:build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": ["npm run build:client", "npm run build:server"],
+        "parallel": true,
+        "cwd": "packages/samples/headless-react"
+      }
+    },
+    "build": {
+      "dependsOn": ["cached:build"],
```

Comment:
Can we mark headless build as a dependency for all samples/headless-* pkgs?

Reply:
Same as above, there's no need since `headless-react-samples:cached:build` already depends on `^build` (by default, inherited from `nx.json`).